resources/assets/js/helpers.js

```diff
@@ -115,8 +141,9 @@ $.extend(true, laravelValidation, {
             var fmt = new DateFormatter();
 
             if ($.type(format) == 'object') {
-                if ('laravelDateFormat' in $.validator.staticRules(format)) {
-                    format = $.validator.staticRules(format).laravelDateFormat[0];
+                var dateRule=this.getLaravelValidation('DateFormat', format)
+                if (dateRule!= undefined) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '!==' and instead saw '!='.](https://www.codacy.com/app/albert-moreno-t/laravel-jsvalidation/file/842620831/issues/source?bid=2225045&fileBranchId=2805859#l145)